# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fwupd/sde-version.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Replace /etc/os-release with T2SDE's own /etc/SDE-VERSION
to be able to start the daemon.

  - NoTag <notag@t2sde.org>

--- fwupd-1.9.10/libfwupd/fwupd-common.c.vanilla	2023-12-24 19:19:53.228630455 +0100
+++ fwupd-1.9.10/libfwupd/fwupd-common.c	2023-12-24 19:22:18.703120775 +0100
@@ -166,20 +166,20 @@
 
 	/* override */
 	if (sysconfdir != NULL) {
-		g_autofree gchar *fn2 = g_build_filename(hostdir, sysconfdir, "os-release", NULL);
+		g_autofree gchar *fn2 = g_build_filename(hostdir, sysconfdir, "SDE-VERSION", NULL);
 		if (g_file_test(fn2, G_FILE_TEST_EXISTS))
 			return g_steal_pointer(&fn2);
 	}
 
 	/* host locations */
 	if (g_strcmp0(sysconfdir, "/etc") != 0) {
-		g_autofree gchar *fn2 = g_build_filename(hostdir, "/etc/os-release", NULL);
+		g_autofree gchar *fn2 = g_build_filename(hostdir, "/etc/SDE-VERSION", NULL);
 		if (g_file_test(fn2, G_FILE_TEST_EXISTS))
 			return g_steal_pointer(&fn2);
 	}
-	fn1 = g_build_filename(hostdir, "/usr/lib/os-release", NULL);
-	if (g_file_test(fn1, G_FILE_TEST_EXISTS))
-		return g_steal_pointer(&fn1);
+	//fn1 = g_build_filename(hostdir, "/usr/lib/os-release", NULL);
+	//if (g_file_test(fn1, G_FILE_TEST_EXISTS))
+	//	return g_steal_pointer(&fn1);
 #endif
 	return NULL;
 }
@@ -253,9 +253,9 @@
  * fwupd_get_os_release:
  * @error: (nullable): optional return location for an error
  *
- * Loads information from the system os-release file.
+ * Loads information from the system SDE-VERSION file.
  *
- * Returns: (transfer container) (element-type utf8 utf8): keys from os-release
+ * Returns: (transfer container) (element-type utf8 utf8): keys from SDE-VERSION
  *
  * Since: 1.0.7
  **/
